test/e2e/helpers.go

```diff
@@ -879,6 +879,7 @@ func InstallHelmChart(ctx context.Context, input clusterctl.ApplyClusterTemplate
 		p := helmGetter.All(settings)
 		valueOpts := &helmVals.Options{}
 		valueOpts.Values = values
+		valueOpts.ValueFiles = valueFiles
```

Comment:
I assume the helm api handles an empty array here gracefully? And/or does the same if we include data in both `Values` and `ValueFiles` properties?

(Wondering if we need to do some input validation sanity-checking...)

Reply:
Yes `MergeValues` will iterate over each `[]string` of values and merge them https://github.com/helm/helm/blob/main/pkg/cli/values/options.go#L41